### DISCUSSION TITLE
Correlate child workflows with parent in C#

### DISCUSSION THREAD

**Discussion by michalaml** [elsa-workflows/elsa-core#4041] [Q&A] [answered]
Is there a way to share parent CorrelationId with child workflows spawned in for with RunWorkflow in C#?
Is it possible? If so, how? Is it not against design of Elsa 2.11.

**Comment by michalaml** [comment, marked as answer]
Have found a way to pass information from parent to child workflows through context.WorkflowExecutionContext.Input which looks like intended way.

**Reply by sfmskywalker** [reply]
Yes, that is the way to go. Perhaps, as this pattern evolves, we might add "official" support for sharing correlation ID values between parent and child workflows. Until then, your current approach is the recommended solution.
